Exercise_A/about.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Email system</title>
+</head>
+<body>
+    <a href="mailto:ycyching0117@gmail.com">Email system</a>
+</body>
+</html>
```

Comment:
Hello Chloe
Great work with this, this works well as when the link is clicked a user would be able to email you